package.json

```diff
@@ -1,21 +1,9 @@
 {
-  "name": "elm-cli-options-parser",
-  "version": "0.0.1",
-  "description": "Parse CLI options declaritively in Elm.",
-  "main": "argv.js",
   "scripts": {
-    "test": "elm-test",
-    "approve": "elm make --docs=documentation.json && cd examples && elm make src/ElmTest.elm && elm make src/Git.elm && elm make src/Graphqelm.elm && elm make src/Simple.elm && elm make src/Validation.elm && elm make src/Curl.elm && elm make src/Grep.elm && cd -",
-    "build": "cd examples && elm make src/*.elm --output src/elm.js && cd -"
-  },
-  "author": "Dillon Kearns",
-  "license": "ISC",
```

Comment:
Would you like me to add some of this metadata back into package.json?

Reply:
The metadata isn't necessary (especially because the license type was incorrect). The changes to this file look good 👍